BXMSDOC-4380-master: user with process-admin role can't see task in Tasks page

kaldesai opened the pull request:
- [Associated JIRA](https://issues.jboss.org/browse/BXMSDOC-4380)
- [Managing & monitoring business processes in RHPAM doc preview](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4380-MMBP-FP/#manage-tasks-con-managing-and-monitoring-processes)

Can you please review and verify the small section which I have added in Chapter 4. Task management?

nmirasch commented on doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc:
```diff
@@ -9,11 +9,27 @@ image::admin-and-config/task-inbox.png[Task inbox]
 
 Users with the `process-admin` role can view and manage all user tasks from the *Tasks* page in {CENTRAL}, located under *Menu* -> *Manage* -> *Tasks*. Users with the `admin` role can access the *Tasks* page but do not have access rights to view and manage tasks by default.
```
@kaldesai Users with the `process-admin` role  --> Process administrators
I try here to avoid talking about process-admin role and then, at the 'Note' part define how the process administrators can be specified, wdyt?

kaldesai replied:
Ack

nmirasch commented on doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc:
```diff
@@ -9,11 +9,27 @@ image::admin-and-config/task-inbox.png[Task inbox]
+    </system-properties>
+----
+=====
+
```
Note:
The tasks which are displayed on the *Menu* → *Manage* → *Tasks* page or *Menu* → *Track* → *Task Inbox* page are determined by the current user permissions.
To be able to manage tasks the user needs to be designed as process administrator, asserting one of this conditions: 
- User is the one specified as 'admin user' (by default 'administrator') 
- User belongs to 'admin group'.(by default 'administrators')
Both values can be configurated via system properties. To change the default value open `standalone-full.xml` from the {PRODUCT} installation directory, set the following property under the <system-properties> tag:  

<system-properties>
    ...
    <property name="org.jbpm.ht.admin.user" value="administrator"/>
    <property name="org.jbpm.ht.admin.group" value="administrators"/>
</system-properties>

nmirasch commented on doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc:
```diff
@@ -9,11 +9,27 @@ image::admin-and-config/task-inbox.png[Task inbox]
+    </system-properties>
+----
+=====
+
```
Note:
The tasks which are displayed on the *Menu* → *Manage* → *Tasks* page or *Menu* → *Track* → *Task Inbox* page are determined by the current user permissions.
To be able to manage tasks the user need to be designed as process administrator, asserting one of this conditions: 
- User is specified as 'admin user' (by default 'administrator') 
- User belongs to 'admin group'.(by default 'administrators')

Both values can be configured via system properties. To change the default value open `standalone-full.xml` from the {PRODUCT} installation directory, set the following property under the <system-properties> tag:  

<system-properties>
    ...
    <property name="org.jbpm.ht.admin.user" value="administrator"/>
    <property name="org.jbpm.ht.admin.group" value="administrators"/>
</system-properties>

nmirasch replied:
@kaldesai Tried to explain the "To assign users to `Administrators` group, open `standalone-full.xml`.." is not correct. The change of the system property is when you want define a different group as 'process administrators'.

kaldesai replied:
Ack

kaldesai commented:
Hey @nmirasch , 
I have incorporated your suggestions in the doc. Instead of Note I have added a paragraph. 
Can you please check and review the content for the same and let me know what do you think. :) Updated link is in description box.
Thank you.

nmirasch commented on doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc:
```diff
@@ -7,13 +7,32 @@ A user task can be assigned to a particular user, multiple users, or to a group.
+----
+<system-properties>
+        ...
+        <property name="org.jbpm.ht.admin.group" value="admin"/>
```
maybe add here  the two properties with the default value?  
    <property name="org.jbpm.ht.admin.user" value="administrator"/>
    <property name="org.jbpm.ht.admin.group" value="administrators"/>

jstastny-cz replied:
+1 for default values here (or we need to comment, that we're actually overriding the group here with value 'admin')

kaldesai replied:
If they are not defined, the values are **Administrator** and **Administrators**. 
With the default distribution, the value is defined as  <property name="org.jbpm.ht.admin.group" value="**admin**"/>
I have already explained about the default values in the description. 

So, me and @nmirasch agreed on keeping that code snippet as it is.

````
<system-properties>
        ...
       * <property name="org.jbpm.ht.admin.user" value="**Administrator**"/>*
       * <property name="org.jbpm.ht.admin.group" value="**Administrators**"/>*
    </system-properties>
````

@nmirasch and @jstastny-cz, what do you think? What should be the correct values in the code snippet?

sterobin commented on doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc:
```diff
@@ -7,13 +7,32 @@ A user task can be assigned to a particular user, multiple users, or to a group.
+* User is specified as `task admin user`. The default value is `Administrator`.
+* User must belong to the task administrators group. The default value is `Administrators`.
+
+Both the values can be configured via `org.jbpm.ht.admin.user` and `org.jbpm.ht.admin.group` system properties.
```
"You can configure the user and user group assignment with the `org.jbpm.ht.admin.user` and `org.jbpm.ht.admin.group` system properties."

Avoid passive, don't use "via" or latinate phrases, and make it clear that each property is for each thing ("both" made it seem like both applied to each, imo)

kaldesai replied:
Noted.

sterobin commented on doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc:
```diff
@@ -7,13 +7,32 @@ A user task can be assigned to a particular user, multiple users, or to a group.
+
+Both the values can be configured via `org.jbpm.ht.admin.user` and `org.jbpm.ht.admin.group` system properties.
+
+To change the default value, open `standalone-full.xml` from the {PRODUCT} installation directory, set the following property under the <system-properties> tag:
```
"... value of these user system properties, open the `standalone-full.xml` file in the {PRODUCT} installation directory, and set the following property under the `<system-properties>` tag:"

Note several corrections in the above rewrite, either for clarity, consistency, or accuracy.

kaldesai commented on doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc:
```diff
@@ -7,13 +7,32 @@ A user task can be assigned to a particular user, multiple users, or to a group.
-Users with the `process-admin` role can view and manage all user tasks from the *Tasks* page in {CENTRAL}, located under *Menu* -> *Manage* -> *Tasks*. Users with the `admin` role can access the *Tasks* page but do not have access rights to view and manage tasks by default.
+Process administrators can view and manage all user tasks from the *Tasks* page in {CENTRAL}, located under *Menu* -> *Manage* -> *Tasks*. Users with the `admin` role can access the *Tasks* page but do not have access rights to view and manage tasks by default.
+
+The tasks which are displayed on the *Menu* → *Manage* → *Tasks* page or *Menu* → *Track* → *Task Inbox* page are determined by the current user permissions.
```
Done

kaldesai commented on doc-content/enterprise-only/admin-and-config/manage-tasks-con.adoc:
```diff
@@ -7,13 +7,32 @@ A user task can be assigned to a particular user, multiple users, or to a group.
+
+The tasks which are displayed on the *Menu* → *Manage* → *Tasks* page or *Menu* → *Track* → *Task Inbox* page are determined by the current user permissions.
+
+To manage all the tasks, user must be specified as a process administrator by defining any of the following conditions:
```
Done
